### DISCUSSION TITLE
A direct link to the location list

### DISCUSSION THREAD

**Discussion by twisteros** [breezy-weather/breezy-weather#1202] [Ideas]
### Describe your idea

The material weather app had one of the headers clickable ie. daily forecast or hourly forecast can't remember exactly. Clicking on that header took user to the location list. It was very handy especially when there were many locations and swiping would be kind of tedious. Can we have such link in Breezy as well? It could be the temperature as it's at the top and quite large. 

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.5 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
In Geometric Weather, you could tap on the "Refreshed at" area, which was always on top of the first card. Taping on the header of any card does nothing.

We moved the Refresh time area on top left to save some space so you can reach the data sooner. I'm not sure opening the location list when tapping on the refresh time makes sense. However, if you move your finger a bit more on the right, you can tap the "3 places" icon to achieve what you are looking for.

**Reply by twisteros** [reply]
I currently have 11 towns I check randomly current weather for on daily basis. I believe there are more people like me. Swiping across locations or going by three dots and then menu item is pretty uncomfortable :( Geometric Weather had two ways to do that. One "Refreshed at" header as you said and the other was a dedicated icon next to the clockwork icon. Obviously making the Refresh time area tappable would do the job without taking any space.

![image](https://github.com/user-attachments/assets/4d928dca-5b4b-4c60-bdda-93471e5c5178)

**Reply by papjul** [reply]
What I mean is the "Refreshed at" area is now close to the other button which does exactly the same thing.
The other reason not to do it is that it's not intuitive and does not make sense to tap on the "Refreshed at" area to open a location list.

![image](https://github.com/user-attachments/assets/6c206d18-ee51-4bb5-8a0f-f45273ccc08f)
